README.md

```diff
@@ -38,6 +41,15 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list. Feel fr
 <br>
 > <h3> Libraries</h3>
 
+#### Loaders
+
+- [Babel Loader](https://github.com/babel/babel-loader) - Webpack plugin for Babel.
+ - Maintainer: `Babel Team`  - [![Github](https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/png/16/github.png)](https://github.com/babel)
```

Comment:
Just pushed a twitter & github SVG into master `./media` I'll do the resize directly in the `<img>` tag